basic/exercise1/script.js

```diff
@@ -1,28 +1,19 @@
 'use strict';
 function formatDate(date) {
+    function insertingZero(i) {
```

Comment:
Please move this function out of formatDate function. If it is nested, then it can be used only by formatDate function. If it is not nested, it can be used in different context also. So in this case it is better not to nest it.